package.json

```
@@ -16,6 +16,9 @@
   },
   "devDependencies": {
     "@ionic/core": "^5.3.2",
-    "@stencil/core": "^2.0.1"
+    "@stencil/core": "^2.0.1",
+    "@types/jest": "^26.0.24",
+    "jest": "^27.0.6",
+    "puppeteer": "~10.0.0"
```

The latest version of Stencil doesn't support v10 of Puppeteer just yet. Although it's landed in the codebase, I can't guarantee a specific date we'll release Stencil v2.7.0 just yet. Would you mind downgrading this in the mean time?
```suggestion
    "puppeteer": "^5.5.0",
    "@types/puppeteer": "^5.4.3"
```
(since we're downgrading puppeteer, we also need to add `@types/puppeteer`)